### PR TITLE
Rin/log4j 2.16

### PR DESCRIPTION
**Pull request by rin-skylight**
## Related Issue or Background Info

- An additional log4j vulnerability was disclosed this week. This PR bumps our version of the affected packages to the latest, which should mitigate any potential denial-of-service attack that can be exploited using version 2.15.0.
- CVE link: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45046
- Additional context: https://www.lunasec.io/docs/blog/log4j-zero-day-update-on-cve-2021-45046/

## Changes Proposed

- Force log4j version 2.16.0. Confirmed using following build scan: https://scans.gradle.com/s/oqezgvoyhdzre/dependencies?dependencies=log4j&expandAll

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "react-redux": "^7.2.1",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^4.0.3",
+    "react-scripts": "^5.0.0",
```
Comment (nickclyde) [inline frontend/package.json:48]: Is this related to the vulnerability? There are definitely breaking changes here.

Comment (nickclyde) [reply frontend/package.json:48]: https://github.com/facebook/create-react-app/releases/tag/v5.0.0

Comment (rin-skylight) [reply frontend/package.json:48]: Apologies, that got slipped in by mistake! I was investigating something the snyk bot was doing earlier. Let me revert that file.

File: frontend/yarn.lock [hunk 2]
```diff
@@ -24,6 +24,15 @@
   dependencies:
```
Comment (zdeveloper) [inline frontend/yarn.lock]: Not sure why the yarn.lock has any changes

**Review by zdeveloper — COMMENTED** [review]
LGTM! Thanks for the quick fix, not sure why the `yarn.lock` was changed

**Review by jdorothy — APPROVED** [review]

**Pull request closed without merging**